src/server/scripts/Kalimdor/HallsOfOrigination/boss_isiset.cpp

```diff
@@ -0,0 +1,858 @@
+    SPELL_MANA_SHIELD_CONTROLLER            = 74382,
+    SPELL_VEIL_OF_SKY_1                     = 74133,
+    SPELL_VEIL_OF_SKY_2                     = 74372,
+    SPELL_VEIL_OF_SKY_3                     = 74373,
+    SPELL_VEIL_OF_SKY_DAMAGE                = 79370,
+
+    // Mirror Images (at 66% and 33%)
+    SPELL_MIRROR_IMAGE_STARRY_SKY_SPAWNER   = 69941,
+    SPELL_MIRROR_IMAGE_STARRY_SKY_N         = 69936,
+    SPELL_MIRROR_IMAGE_STARRY_SKY_E         = 69939,
+    SPELL_MIRROR_IMAGE_STARRY_SKY_W         = 69940,
+    SPELL_ASTRAL_SHIFT                      = 74312,
+    SPELL_ASTRAL_SHIFT_EXPLOSION            = 74333,
+    SPELL_MIRROR_IMAGE_SPAWNER              = 74264,
+    SPELL_MIRROR_IMAGE_N                    = 74263,
+    SPELL_MIRROR_IMAGE_E                    = 74262,
+    SPELL_MIRROR_IMAGE_W                    = 74261,
+    SPELL_ADDS_STATE_VISUAL                 = 74289,
+    SPELL_STARFALL_STATE_VISUAL             = 74265,
+    SPELL_MANA_SHIELD_STATE_VISUAL          = 74266,
+    SPELL_IMAGE_EXPLOSION                   = 74301,
+
+    // Celestial Familiar npc
+    SPELL_ORB_VISUAL                        = 74356, 
+    SPELL_ARCANE_BARRAGE                    = 74374,
+
+    // Starry Sky npc
+    SPELL_STARRY_SKY_VISUAL                 = 74149,
+
+    // Astral Shift Explosion Visual npc
+    SPELL_ASTRAL_SHIFT_EXPLOSION_VISUAL     = 74331,
+
+    // Spatial Flux & Energy Flux (heroic only)
+    SPELL_CALL_OF_SKY                       = 90750  // Summons Spatial Flux (npc script in halls_of_oriignation.cpp)
+};
+
+enum NPCs
+{
+    NPC_CELESTIAL_FAMILIAR                  = 39795,
+    NPC_ASTRAL_RAIN                         = 39720, // N
+    NPC_CELESTIAL_CALL                      = 39721, // E
+    NPC_VEIL_OF_SKY                         = 39722, // W
+    NPC_ISISET_SPATIAL_FLUX                 = 48707  // heroic only
+};
+
+enum Texts
+{
+    SAY_AGGRO                               = 0,
+    SAY_SUPERNOVA                           = 1,
+    SAY_SUPERNOVA_WARNING                   = 2,
+    SAY_PLAYER_KILL                         = 3,
+    SAY_DEATH                               = 4
+};
+
+enum Events
+{
+    EVENT_SUPERNOVA = 1,
+    EVENT_ASTRAL_RAIN,
+    EVENT_ASTRAL_FAMILIAR,
+    EVENT_VEIL_OF_SKY,
+    EVENT_IMAGES_ATTACK,
+    EVENT_DESPAWN_IMAGES,
+
+    EVENT_ORB_ADD_VISUAL,
+    EVENT_ORB_SET_AGGRESSIVE,
+    EVENT_ORB_ARCANE_BARRAGE,
+
+    EVENT_IMAGE_ASTRAL_RAIN_ABILITY,
+    EVENT_IMAGE_ARCANE_BARRAGE_ABILITY,
+    EVENT_IMAGE_VEIL_OF_SKY_ABILITY,
+
+    EVENT_STARRY_SKY_ADD_VISUAL,
+};
+
+enum Actions
+{
+    ACTION_IMAGES_SET_AGGRESSIVE,
+    ACTION_MIRROR_IMAGE_DIED,
+    ACTION_IMAGES_SET_PASSIVE,
+    ACTION_IMAGES_DESPAWN
+};
+
+class boss_isiset : public CreatureScript
+{
+public:
+    boss_isiset() : CreatureScript("boss_isiset") { }
+
+    struct boss_isisetAI : public BossAI
+    {
+        boss_isisetAI(Creature* creature) : BossAI(creature, DATA_ISISET), _transitionPhase(false) { }
```

Comment:
```
Member variable 'boss_isisetAI::_mirrorImageDied' is not initialized in the constructor.
```
https://app.codacy.com/app/TrinityCore/TrinityCore/pullRequest?prid=1497760
(... or is this a false positive?)